components/src/core/Drawer/DrawerHeader.tsx

```diff
@@ -213,6 +213,7 @@ const DrawerHeaderRender: React.ForwardRefRenderFunction<unknown, DrawerHeaderPr
                                     onIconClick();
                                 }}
                                 edge={'start'}
+                                size="large"
```

Comment:
This line updated because of preset-mode. We do not need this line so, removed it. It didn't effect drawerHeader since it works only with iconButton's fontSize prop. 

Thanks for observing this.